### PR TITLE
Rename callbacks

### PR DESCRIPTION
**Pull request by tompsota**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# freeRASP 3.5.0
+
+- ⚠️ Updated the `talsecStart()` method to return `Promise<string>`. If freeRASP starts successfuly, the method will return `freeRASP started` string. There are not any breaking changes if you are using the provided `useFreeRasp` hook.
```
Comment (xprikryl2) [inline CHANGELOG.md:3]: Isn't this an API change? Or is it backward compatible? If it's not backward compatible, we should increase the library's major version.

File: android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt [hunk 2]
```diff
@@ -125,5 +117,24 @@ class FreeraspReactNativeModule(val reactContext: ReactApplicationContext) :
 
   companion object {
     const val NAME = "FreeraspReactNative"
+    val THREAT_CHANNEL_NAME = (10000..999999999).random()
+      .toString() // name of the channel over which threat callbacks are sent
+    val THREAT_CHANNEL_KEY = (10000..999999999).random()
+
+      .toString() // key of the argument map under which threats are expected
```
Comment (xprikryl2) [inline android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt:124]: This is strange formatting.

File: CHANGELOG.md (hunk 1 above)
Comment (tompsota) [reply CHANGELOG.md:3]: It is backward compatible, just the return value will be ignored

File: android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt (hunk 2 above)
Comment (tompsota) [reply android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt:123]: fixed

**Pull request merged**